src/scanoss/agent/external/inc/json.h

```diff
@@ -0,0 +1,281 @@
+
```

Comment:
Can we use the [json-c](https://github.com/json-c/json-c) instead? It is already used in [nomos to create JSON output](https://github.com/fossology/fossology/blob/master/src/nomos/agent/json_writer.c)

It will be more cleaner and easier to maintain.

Reply:
Hi @GMishx, This file is part of our original Scanner CLI source code (https://github.com/scanoss/scanner.c). Let me check if we can publish a modified version.

Reply:
Hi @GMishx. This file is part of our original Scanner CLI (https://github.com/scanoss/scanner.c) and it could be updated every time we release a new version of it. Let me check with the team if we can publish a modified version of the Scanner.